### PR TITLE
data: chirag4242

### PR DESCRIPTION
**Pull request by chirag4242**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6417"></a>

### CODE REVIEW

**Review by Arindam200 — DISMISSED** [review]

File: data/chirag4242.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+  "name": "Chirag Patil",
+  "type": "personal",
+  "bio": "I am a fullstack software developer",
+  "avatar": "https://github.com/chirag4242.png",
```
Comment (imbeshat) [inline data/chirag4242.json:5]:
```suggestion
```

Comment (imbeshat) [reply data/chirag4242.json:5]: not required

File: data/chirag4242.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "ReactJs",
+    "C#",
+    "Python"
```
Comment (imbeshat) [inline data/chirag4242.json:12]:
```suggestion
    "Python",
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

File: data/chirag4242.json (hunk 1 above)
Comment (chirag4242) [reply data/chirag4242.json]: Just an avatar right? fixed that

File: data/chirag4242.json (hunk 2 above)
Comment (chirag4242) [reply data/chirag4242.json]: fixed that also 👍

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**